lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -4412,6 +4412,30 @@ would appear as ``$options.vcf_output`` having the values ``'--vcf'`` when true
       </data>
     </outputs>
 ```
+
+Variables that have the type attribute ``select`` need a special python statement of
+the form ``selection and "option" in selection`` inside the ``<filter>`` tags. 
+
+### Example
+
+```xml
+    <inputs>
+       <param name="outputs" type="select" optional="true">
+            <option value="save_phase">Phase Movie</option>
+            <option value="save_period">Period Movie</option>
+        </param>
+    </inputs>
+
+    <outputs>
+        <data name="phase_out" format="tiff" label="${movie.name[:-4]}_phase">
+            <filter>outputs and "save_phase" in outputs</filter>
+        </data>
+        <data name="period_out" format="tiff" label="${movie.name[:-4]}_period">
+            <filter>outputs and "save_period" in outputs</filter>
+        </data>
```

Comment:
```suggestion
...
```

I think one is sufficient for the example

Reply:
I thought 2 makes sense, as for one alone a boolean is enough?

Reply:
An alternative would be to link to an existing tool instead of giving the example here. This is also often done in this file. My intention was to keep it short. But I'm also fine with 2.

Reply:
@bernt-matthias we will use this documentation more often in editors and interactive hints. I guess there is some advantage in the future to have small examples as part of the documentation.

Reply:
e.g. in https://github.com/galaxyproject/galaxy-language-server